uploader.json

```diff
@@ -2810,6 +2810,11 @@
                 "reserved_port/range"
             ],
             "symbolic_name": "sap_host_profile"
+        },
+        {
+            "file": "/sys/module/nvme_core/parameters/io_timeout",
+            "pattern": [],
+            "symbolic_name": "nvme_core_io_timeout"
```

Comment:
@lonicerae please move this spec up so that the symbolic name is in sorted, alphabet order.

Reply:
Okay, got it.